V1 docs (#111)
NativePHP/nativephp.com: resources/views/docs/mobile/1/getting-started/installation.md

## Requirements

1. PHP 8.3+
2. Laravel 10 or higher
3. [A NativePHP for mobile license](https://checkout.anystack.sh/nativephp-ios)
6. _Optional_ iOS/Android device

#### For iOS
1. An Apple Mac (ideally Silicon) running macOS 12+ with Xcode 16+ 
2. An active [Apple Developer account](https://developer.apple.com/)
3. You can download Xcode from the Mac App Store.

#### For Android
1. [Android Studio Giraffe (or later)](https://developer.android.com/studio)
2. The following environment variables set.
3. You should be able to successfully run `java -v` and `adb devices` from the terminal.
4. **Windows only**: You must have [7zip](https://www.7-zip.org/) installed.

#### For macOS
```shell
export JAVA_HOME=$(/usr/libexec/java_home -v 17)
export ANDROID_SDK_ROOT=$HOME/Library/Android/sdk
export PATH=$PATH:$JAVA_HOME/bin:$ANDROID_HOME/emulator:$ANDROID_HOME/tools:$ANDROID_HOME/tools/bin:$ANDROID_HOME/platform-tools
```

#### For Windows
```shell

> set JAVA_HOME=C:\Program Files\Microsoft\jdk-17.0.8.7-hotspot

Reviewer: I'm wondering if this can be replaced with a programatic approach? I'm not sure everyone will have 17.0.8.7 specifically? Not a blocker, just an observation.

Is it possible this isn't required if JAVA_HOME is already set in the Windows Env Variables?

set ANDROID_SDK_ROOT=C:\Users\yourname\AppData\Local\Android\Sdk
set PATH=%PATH%;%JAVA_HOME%\bin;%ANDROID_SDK_ROOT%\platform-tools
```

> **Note** You cannot build iOS apps on Windows or Linux

You don't _need_ a physical iOS/Android device to compile your application and test it for your app, as NativePHP for mobile supports
the iOS Simulator and Android emulators. However, we highly recommend that you test your application on a real device before submitting to the
App/Google Play Store.

## Laravel

NativePHP for mobile is built to work with Laravel. You can install it into an existing Laravel application, or
[start a new one](https://laravel.com/docs/installation). The most painless way to get PHP and Node up and running on your system is with
[Laravel Herd](https://herd.laravel.com). It's fast and free!

> ## Private package

Reviewer 2: I think I will merge the `Private package` and `Install NativePHP for mobile` sections.

To make NativePHP for mobile a reality has taken a lot of work and will continue to require even more. For this reason,
it's not open source, and you are not free to distribute or modify its source code.

Before you begin, you will need to purchase a license.

> Licenses can be obtained via [Anystack](https://checkout.anystack.sh/nativephp-ios).

Reviewer: I believe this link is invalid as @simonhamp has merged in the PR to bypass Anystack Billing.

Would be good to have @simonhamp input here.

Reviewer 3: Pete is correct here, but I would proceed with this PR as is and I can modify the billing PR to find other checkout.anystack.sh urls throughout the repo. 👍

Reviewer 4: I already fixed these on `main` yesterday when we switched the URL

Author: This is fixed

## Requirements

1. PHP 8.3+
2. Laravel 10 or higher
3. An Apple Silicon Mac running macOS 12+ with Xcode 16+
4. An active [Apple Developer account](https://developer.apple.com/)
5. [A NativePHP for mobile license](https://checkout.anystack.sh/nativephp)
6. _Optional_ iOS device

#### For iOS
1. An Apple Mac (ideally Silicon) running macOS 12+ with Xcode 16+ 
2. An active [Apple Developer account](https://developer.apple.com/)
3. You can download Xcode from the Mac App Store.

#### For Android
1. [Android Studio Giraffe (or later)](https://developer.android.com/studio)
2. The following environment variables set.
3. You should be able to successfully run `java -v` and `adb devices` from the terminal.
4. **Windows only**: You must have [7zip](https://www.7-zip.org/) installed.

#### For macOS
```shell

> export JAVA_HOME=$(/usr/libexec/java_home -v 17) // This isn't required if JAVA_HOME is already set in the Windows Env Variables

Reviewer: ```suggestion
export JAVA_HOME=$(/usr/libexec/java_home -v 17) // This isn't required if JAVA_HOME is already set in your system Environment Variables.
```

I believe it's the same for Mac, so likely no need to reference Windows specifically. :)
